src/components/Forms/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,94 @@
+import { Button, DialogActions, TextField } from "@mui/material";
+import { useCallback, KeyboardEvent, useState, useMemo } from "react";
+import {
+  passwordValidationError,
+  passwordValidationHelper,
+  usernameValidationError,
+  usernameValidationHelper,
+} from "../formValidation.utils";
+
+interface ILoginFormProps {
+  onSubmit: (username: string, password: string) => void;
+}
+
+const LoginForm = (props: ILoginFormProps) => {
+  const { onSubmit } = props;
+
+  const [username, setUsername] = useState<string>("");
```

Comment:
if you set the default value to string and it will only be string you don't need to explicitly specify the type here
```suggestion
  const [username, setUsername] = useState("");
```

Reply:
Fixed